articles/arm-x86/index.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/arm-x86/
+title: ARM vs x86 - Explained
+description: 
+author: mike-white
+date: 2020-09-10T00:00:00-11:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/arm-x86/hero.jpg
+    alt: 
+---
+Recently, Apple [announced their decision to switch from Intel CPUs to their own ARM chips](https://www.theverge.com/2020/6/22/21295475/apple-mac-processors-arm-silicon-chips-wwdc-2020). What's the difference? The new Apple Silicon chips are based on ARM CPUs, like the CPUs for iPhone and iPad. The Intel chips use Intel's proprietary x86 architecture.
```

Comment:
Solid hook. However, the distinction is not clear enough.

"like the CPUs for iPhone and iPad" => these used Intel's chips before though, no? Including this is confusing. Why was ARM preferred as an architecture over Intel for its iPhone and iPads? This is what you need to point out.

Reply:
That's incorrect. The very first iPad used Apple's own ARM processors. The first iPhone used ARM processors made by Samsung, but they switched to Apple's own CPUs with the iPhone 4.